front/src/containers/Profile/Config/index.js

```diff
@@ -0,0 +1,56 @@
+// react imports
+import React from 'react'
+
+import {
+  View,
+  ScrollView
+} from 'react-native'
+
+// third imports
+import SimpleLineIcons from 'react-native-vector-icons/SimpleLineIcons'
+import MaterialIcons from 'react-native-vector-icons/MaterialIcons'
+import Octicons from 'react-native-vector-icons/Octicons'
```

Comment:
deixa como os outros: **OctionsIcons**. Igual pra Feather => **FeatherIcons**

Reply:
I use the name of final in import